release.mk

```diff
@@ -39,7 +39,7 @@ PROJECT_PATCH_VERSION ?= $(shell echo $(RELEASE_VERSION) | cut -f3 -d.)
 PROJECT_OWNER ?= elastic
 RELEASE_TYPE ?= minor
 
-CURRENT_RELEASE ?= $(shell gh api repos/elastic/apm-server/releases/latest | jq -r '.tag_name|sub("v"; ""; "")')
+CURRENT_RELEASE ?= $(shell gh release list --exclude-drafts --exclude-pre-releases --limit 10 --json tagName --jq '.[].tagName|select(. | startswith("v$(PROJECT_MAJOR_VERSION)"))' | sed 's|v||g' | sort -r | head -n 1)
```

Comment:
We cannot use `latest` as long as latest can point to `7.17`. Already reported to the release team. 

<img width="754" alt="image" src="https://github.com/elastic/apm-server/assets/2871786/9f241c95-482b-40e1-a225-5a2a834a34fd">


I decided to use a different approach and filter by the major version and do some manipulation